tests/unit/test_deprecations.py

```diff
@@ -41,11 +41,11 @@ def test_gold_method(self):
     def test_reddit_token_manager(self):
         with pytest.raises(DeprecationWarning):
             Reddit(
+                token_manager=FileTokenManager("name"),
```

Comment:
This sort is not in the right place, if that was the intent, otherwise, I'm not sure why this was moved here and in a few other places.

Reply:
See above.